ParisBNF/abb/BNFabb207.xml

```diff
@@ -0,0 +1,278 @@
+                            </msItem>
+
+                                <msItem xml:id="p1_i2">
+                           <locus from="33r"/>
+                             <title type="complete" ref="LIT2807RepCh91"/>
+                             <textLang mainLang="gez"/>
+                                    <incipit xml:lang="gez">ሰላም፡ ለኪ፡ ማርያም፡ ድንግል፡ ዘመዐዛ፡ አፉኪ፡ ኮል</incipit>
+                                </msItem>
+
+                                <msItem xml:id="p1_i3">
+                            <locus from="34v"/>
+                                    <title type="complete" ref="LIT7021BaryaAganentGudale"/>
+                              <textLang mainLang="gez"/>
+                              <incipit xml:lang="gez">በስመ፡ አብ፡ <gap reason="ellipsis" resp="CR"/>ዛሙስ፡ የአድ፡ ታውሚ፡ አስሆሩ<gap reason="illegible" unit="chars" quantity="1" resp="CR"/>ብለባርያ፡ 
+                                  አገባጅ፡ ዘበላዕለ፡ ምሻቱር፡ ሰሎሞን፡ አርምኃ። <gap reason="ellipsis" resp="CR"/>ወለአጋንንት፡ ለደስክ፡ ወለጉዳሌ፡ ለቡዳ፡ ወለተግባረ፡ ሰብእ፡ ለውግዓት፡ 
+                                  ወለቁርጥማ፡ ለድድቅ፡ ወለጋኔነ፡ ቀትር፡ ለፌራ፡ ወለዳስማ፡ ለምትአት፡ ወለፅፋፍአት፡</incipit>
+                         </msItem>
+
+                                <msItem xml:id="p1_i4">
+                            <locus from="38r"/>
+                                    <title type="incomplete" ref="LIT7022HemamaBedbed">ጸሎት፡ በእንተ፡ ሕማመ፡ ብድብድ፡</title>
+                             <textLang mainLang="gez"/>
+                                    <note>How the Lord spoke to ʾAmru at Mirod, he sent to ʾAmru an angel called Ṭǝqonṭoqyā, 
+                                    </note>
+                                </msItem>                              
+                        </msContents>
+                                     
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="parchment"/>
+                                    </support>
+                                    <extent>
+                                        <measure unit="leaf" quantity="40">40</measure>
+                                        <locus from="1" to="40"/>
+                                    </extent>
+                                </supportDesc>
+                                <layoutDesc>
+                                    <layout columns="2" writtenLines="10"/> 
+                                </layoutDesc>                       
+                                </objectDesc>
+                                
+                                <handDesc>
+                                    <handNote xml:id="h1" script="Ethiopic" corresp="#p1 #p2_i2">                                        
+                                        <seg type="script">18th-century script.</seg>                                        
+                                    </handNote>
```

Comment:
Is it useful to provide a date in `<date notBefore="1700" notAfter="1799" resp="DN"/>` in addition to
 `<seg type="script">18th-century script.</seg>` or is it redundant?

Reply:
I used to shape this segment in this way; here it is simple, but most frequently, concerning the script, we must sway away from exact numbers (if we say "late", "close to the end", "early", "around the middle"). If you think it is importan, please open an issue.

Reply:
I just wanted to know, whether <date/> is neccessary to make the hands searchable. I will open an issue to clarify this question.